dgp/gui/workspaces/TransformTab.py

```diff
@@ -105,101 +126,113 @@ def _segments(self) -> List[DataSegmentController]:
         return [self._dataset.segment_model.item(i)
                 for i in range(self._dataset.segment_model.rowCount())]
 
-    def _auto_range(self):
-        """Call autoRange on all plot surfaces to scale the view to its
-        contents"""
-        for plot in self.plot.plots:
-            plot.autoRange()
-
-    def _view_transform_graph(self):
-        """Print out the dictionary transform (or even the raw code) in GUI?"""
-        pass
-
-    def _set_mask(self):
-        # TODO: Decide whether this is useful to allow viewing of a single line
-        # segment
-        pass
+    def _graph_source(self, index):  # pragma: no cover
+        """Utility to display the transform graph source (__init__) method
+        containing the definition for the graph.
 
-    def _clear_mask(self):
-        pass
+        If Pygments is available the source code will be highlighted
 
-    def _split_by_segment(self, segments: List[DataSegmentController], series):
+        Notes
+        -----
+        The inspection of the source code is somewhat fragile and dependent on
+        the way the graph is defined in the source. The current method gets the
+        __init__ source code for the TransformGraph descendant then searches for
+        the string index of 'self.transform_graph', and takes from the first '{'
+        until the first '}'.
```

Comment:
We will soon be able to overhaul this soon with the TransformGraph revamp, but it's good to see what you are doing here so I can provide a better way of doing this. Let's talk online more about what you're thinking here in terms of functionality.